app/views/admin/custom_emojis/index.html.haml

```diff
@@ -72,7 +72,7 @@
             .input.select.optional
               .label_input
                 = f.select :category_id,
-                           options_from_collection_for_select(CustomEmojiCategory.all, 'id', 'name'),
+                           options_from_collection_for_select(CustomEmojiCategory.order(:name).all, 'id', 'name'),
```

Comment:
Minor style nitpicks:

- Use `order(name: :asc)` to be explicit about what we want
- Add an `alphabetic` (or something) scope on the model to capture the order (mirrors an existing scope on `CustomEmoji`)
- Move the entire `CustomEmojiCategory...all` to a helper method, to avoid directly referring to a constant from the view (this isn't something you added, it's already like that, but now may be good time to extract)

Reply:
Unsure about the 3rd pattern, but i'll move this to an `.alphabetic` scope on the model, and then add `.all`

If you have an example of a helper method, please show it (unless its just `alphabetic.all`), and help me think of a proper name for it. It seems this function just accepts an activerecord query/collection, and turns it into options on its own

Reply:
Applied 1 and 2, say if this is enough, or if you want 3 as well (and then please give me a place/name that i could use for it, or an example i can follow)